README.md

```diff
@@ -7,8 +7,38 @@ with your computer.
 
 ### :hammer_and_wrench: How to compile the desktop client
 
-- Instructions for compiling the client on Linux, Windows and mac OS can be found at:
-[Wiki/How to compile the desktop client](https://github.com/nextcloud/desktop/wiki/How-to-compile-the-desktop-client).
+#### :memo: Step by step instructions
+
```

Comment:
Maybe we should add a link to the dependencies required to build here as well: 
https://github.com/nextcloud/desktop/wiki/System-requirements-for-compiling-the-desktop-client